portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,368 @@
+}
+.column {
+  display: table-cell;
+  text-align: center;
+}
+
+.rowImage {
+  width: 15vw;
+  height: 30vh;
+}
+
+.topnav {
+  margin-bottom: 0; 
+  overflow: hidden;
+  background-color: rgb(231, 131, 16);
+  position: fixed; 
+  width: 360px;
+  height: 45px;
+  border-radius: 50px;
+  text-align: center; 
+}
+
+.topnav a {
+  color: #f2f2f2;
+  text-align: center;
+  padding: 14px 16px;
+  text-decoration: none;
+  font-size: 22px;
+  border-radius: 50px;
+}
+
+.topnav a:hover {
+  background-color: #ebf2f8;
+  color: black;
+}
+
+.extendedBar {
+  width: 100%; 
+}
+
+
+.sectionHeader {
+  padding-top: 15px;
+  text-align: center;
+  font-size: 40px; 
+  margin-bottom: 1px;
+}
+
+.round {
+  border-top: 8px solid rgb(231, 131, 16);
+  border-radius: 5px;
+  width: 250px;
+  margin-top: 0px;
+}
+
+.sectionRow {
+  padding-top: 70px;
+}
+
+.aboutMeSection {
+  padding-top: 14px;
+}
+
+.profile {
+  vertical-align: middle;
+  width: 169px;
+  height: 191px;
+  border-radius: 50%;
+}
+
+.fishingPhoto {
+  width: 170px;
+  height: 192px;
+  border-radius: 50%; 
+}
+
+.button {
+  background-color: rgb(231, 131, 16);
+  border: 2px solid rgb(231, 131, 16);
+  color: white;
+  padding: 5px;
+  text-align: center;
+  text-decoration: none;
+  font-size: 13px;
+  margin: 7px 2px;
+  transition-duration: 1s; 
+  border-radius: 5px;
+}
+
+.button:hover {
+  background-color: rgb(244, 247, 244);
+  color: rgb(231, 131, 16); 
+}
+
+#myBtn {
+  display: none;
+  position: fixed;
+  bottom: 20px;
+  right: 30px;
+  z-index: 99;
+  font-size: 18px;
+  border: none;
+  outline: none;
+  background-color: rgb(231, 131, 16);
+  color: white;
+  cursor: pointer;
+  padding: 15px;
+  border-radius: 4px;
+}
+
+#myBtn:hover {
+  background-color: rgb(244, 247, 244);
+  color: rgb(231, 131, 16);
+}
+
+#eduButton {
+  background-color: rgb(231, 131, 16);
+  border: 2px solid rgb(231, 131, 16);
+  color: white;
+  padding: 5px;
+  text-align: center;
+  text-decoration: none;
+  font-size: 13px;
+  margin: 7px 2px;
+  transition-duration: 1s; 
+  border-radius: 5px;
+}
+
+#eduButton:hover {
+  background-color: rgb(244, 247, 244);
+  color: rgb(231, 131, 16);
+}
+
+#interestsButton {
+  background-color: rgb(231, 131, 16);
+  border: 2px solid rgb(231, 131, 16);
+  color: white;
+  padding: 5px;
+  text-align: center;
+  text-decoration: none;
+  font-size: 13px;
+  margin: 7px 2px;
+  transition-duration: 1s; 
+  border-radius: 5px;
+}
+
+#interestsButton:hover {
+  background-color: rgb(244, 247, 244);
+  color: rgb(231, 131, 16);
+}
+
+#funFactsButton {
+  background-color: rgb(231, 131, 16);
+  border: 2px solid rgb(231, 131, 16);
+  color: white;
+  padding: 5px;
+  text-align: center;
+  text-decoration: none;
+  font-size: 13px;
+  margin: 7px 2px;
+  transition-duration: 1s; 
+  border-radius: 5px;
+}
+
+#funFactsButton:hover {
+  background-color: rgb(244, 247, 244);
+  color: rgb(231, 131, 16);
+}
+
+#mobileButton {
+  background-color: rgb(231, 131, 16);
```

Comment:
Optional: I think you can declare this rgb value as a variable and reuse it everywhere.

See https://www.w3schools.com/css/css3_variables.asp